tests/zino_test.py

```diff
@@ -269,3 +269,13 @@ def test_when_pysnmp_is_backend_it_should_not_log_low_level_details(self, state_
                 with caplog.at_level(logging.DEBUG):
                     zino.log_snmp_session_stats()
                     assert "gc reachable (low-level)=" not in caplog.text
+
+    def test_when_debug_logging_is_not_enabled_it_should_not_log_anything(self, state_with_localhost, caplog):
+        import zino.snmp.netsnmpy_backend as backend
+
+        with patch.object(zino.state, attribute="state", new=state_with_localhost):
+            with patch.object(zino, attribute="import_snmp_backend") as import_snmp_backend:
+                import_snmp_backend.return_value = backend
+                with caplog.at_level(logging.INFO):
+                    zino.log_snmp_session_stats()
+                    assert "gc reachable" not in caplog.text
```

Comment:
Shouldn't this be in the commit that introduces the logging?

Reply:
Yes, this must be a fixup commit gone wrong :P